src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -15,7 +17,7 @@ import java.io.FileWriter
 
 
 data class HaplotypeSequence(val id: String, val sequence: String, val refRangeId: String, val refContig: String, val refStart: Int, val refEnd: Int,
```

Comment:
Should this have a checksum?

Reply:
The checksum is the id.  So it does not need it as it would just be duplicated.